pkg/services/dashboards/dashboard.go

```diff
@@ -36,6 +33,13 @@ type DashboardService interface {
 	GetSoftDeletedDashboard(ctx context.Context, orgID int64, uid string) (*Dashboard, error)
 }
 
+type DashboardReadService interface {
```

Comment:
Do you mind making a PR that has just this change and the the usage of it where possible?

(unless it is really just a couple files)

Reply:
Updated to remove this definition for now - will follow up with a separate PR